bornhack/templates/info.html

```diff
@@ -143,10 +143,28 @@ <h4 class="panel-title">Public Transportation</h4>
             <h4>From Abroad</h4>
             <p>If you are going to BornHack from abroad you have different options as well.</p>
             <dl>
+             <dt>Berlin (Germany)</dt>
+              <dd>
+                There is a direct train to Sassnitz. From Sassnitz next section.</br>
```

Comment:
This is not correct. The [ferry](http://www.faergen.com/routes/bornholmerfaergen/sassnitz.aspx) is 4km away from Sassnitz train station. You'd need to travel to "Sassnitz-Mukran Fährhafen" instead of "Sassnitz".